oarepo_requests/services/ui_schema.py

```diff
@@ -164,19 +166,21 @@ def _get_stateful_labels(
         try:
             topic = ResolverRegistry.resolve_entity(data["topic"], False)
             if topic:
+                request_obj = None
```

Comment:
Need to pass fully-fledged request, not just a projection, as the stateful name might need creator, recipient, ...